TACC/Core-Portal
# task/FP-1886: Remove My Account from pulldown at the top right

## Overview
Hiding the "My Accounts" button in the nav drop-down if the hideManageAccount setting is True


## Related

* [FP-1886](https://jira.tacc.utexas.edu/browse/FP-1886)

## Changes
If the `WORKBENCH_SETTINGS` boolean `hideManageAccount` is set to true, it will now hide the "My Accounts" button in the top-right drop down nav. This means that `WORKBENCH_SETTINGS` is now passed to this Django template as a context variable.


## Testing

1. In the `settings_custom.py` file, make sure `hideManageAccount` is set to `True` in the `_WORKBENCH_SETTINGS` dictionary
2. Navigate to any page in portal, and check if the "My Account" button is available under the top-right nav dropdown. It should be hidden. (See UI below)
3. Set `hideManageAccount` to `False`
4. The "My Account" button should now be available under the top-right nav dropdown.

## UI
![Screen Shot 2022-10-20 at 5 08 03 PM](https://user-images.githubusercontent.com/7462330/197068557-0f185a5c-1087-4197-9df2-6e2548050b35.png)


## Notes
